### ISSUE TITLE
Cannot load libv8executor.so due to missing symbol

### ISSUE THREAD

**Issue by hoangpham95** [Kudo/react-native-v8#54]
I am trying to build my application with react-native-v8 (RN 0.62.2) and got this error
```java
2020-08-22 02:30:30.085 8421-8492/? D/SoLoader: About to load: libv8executor.so
2020-08-22 02:30:30.085 8421-8492/? D/SoLoader: libv8executor.so not found on /data/data/[redacted]/lib-main
2020-08-22 02:30:30.085 8421-8492/? D/SoLoader: libv8executor.so found on /data/app/[redacted]==/lib/x86
2020-08-22 02:30:30.085 8421-8492/? D/SoLoader: Not resolving dependencies for libv8executor.so
java.lang.UnsatisfiedLinkError: couldn't find DSO to load: libv8executor.so caused by: dlopen failed: cannot locate symbol "_ZN2v82V810InitializeEv" referenced by "/data/app/[redacted]==/lib/x86/libv8executor.so"... result: 0
        at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:896)
        at com.facebook.soloader.SoLoader.loadLibraryBySoNameImpl(SoLoader.java:725)
        at com.facebook.soloader.SoLoader.loadLibraryBySoName(SoLoader.java:649)
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:629)
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:577)
        at com.facebook.v8.reactexecutor.V8Executor.<clinit>(V8Executor.java:16)
        at com.facebook.v8.reactexecutor.V8ExecutorFactory.create(V8ExecutorFactory.java:20)
        at com.facebook.react.ReactInstanceManager$5.run(ReactInstanceManager.java:997)
        at java.lang.Thread.run(Thread.java:764)
```
I'm using `com.facebook.soloader:soloader:0.9.0` and have android ndk-21 installed.

It looks like the SoLoader can still find the libv8executor.so but it's looks like the libv8executor cannot reference `v8::v8::Initialize()`. Do you know if this is an existing issue, and if yes, how could I resolve this?

**Comment by Kudo** [comment]
Could you please provide your exact version of `react-native-v8` and `v8-android-*`?

**Comment by hoangpham95** [comment]
I'm running 0.62.2-patch.1 for react-native-v8 and 8.84.0 for v8-android.

**Comment by hoanglam10499** [comment]
I have the same issues with version 0.62.2

**Comment by Kudo** [comment]
Please try to upgrade `react-native-v8@0.62.2-patch.2`
0.62.2-patch.1 is not compatible with v8-android 8.80.

There is a quick question.
Do you have v8-android-* explicitly listed in project's package.json?

**Comment by hoangpham95** [comment]
I do have v8-android-* explicitly listed as a dependency. I've upgraded by RN app to 0.63.2 and use the 0.63.2 patch instead. It's working as expected.

**Comment by Kudo** [comment]
Thanks for your answer. Since V8's ABI is not stable between releases. Versioning between react-native-v8 and v8-android-* needs to be coupled. The way to manage the versioning is difficult
Thinking about if `peerDependencies` help to prevent for this case. 🤔

**Comment by aganov** [comment]
`0.61.5-patch.3` is incompatible with v8 `8.84.0`. Downgrading to `8.80.1` solves the issue.

**Comment by Kudo** [comment]
@aganov I am sorry for late. The `react-native-v8@0.61.5-patch.4` is published to support v8-android 8.84.
Please take a look if you still on 0.61.5.

**Comment by Kudo** [comment]
Going to close this issue as there are no further response. Please feel free to reopen or open new one if there are any other problems.
